src/nft-card.ts

```diff
@@ -131,24 +131,36 @@ export class NftCard extends LitElement {
       ? this.contractAddress
       : this.tokenAddress
 
-    let vertCardWidth
-    if (
-      window.innerWidth < MOBILE_BREAK_POINT &&
-      this.orientationMode === OrientationMode.Auto
-    ) {
-      vertCardWidth = VERT_CARD_WIDTH_MOBILE
+    /* If user sets any style overrides assume manual mode unless user has defined the mode */
+    if (!this.orientationMode) {
+      this.orientationMode =
+        this.width || this.height || this.horizontal || this.vertical
+          ? OrientationMode.Manual
+          : OrientationMode.Auto
+    }
+
+    this.horizontal = this.horizontal || !(this.vertical || this.horizontal)
+
+    let vertCardWidth = ''
```

Comment:
why change this to `''` instead of `undefined` like before? confused what the type of the `this.width` should be